templates/form/textInput.tpl

```diff
@@ -43,7 +43,7 @@
 					{if $FBV_readonly} readonly="readonly"{/if}
 					value="{$FBV_value[$thisFormLocale]|escape}"
 					name="{$FBV_name|escape}[{$thisFormLocale|escape}]"
-					id="{$FBV_id|escape}-{$thisFormLocale|escape}{$uniqId}"
+					id="{$FBV_id|escape}-{$thisFormLocale|escape|escape:jqselector}{$uniqId}"
```

Comment:
This double-escaping is messy -- since I think all applications of the `jqselector` escape mode are the same, I would suggest having that one call handle all escaping requirements. (Here and elsewhere.)